api/repositories/service_binding_repository_test.go

```diff
@@ -478,7 +479,7 @@ var _ = Describe("ServiceBindingRepo", func() {
 		})
 	})
 
-	Describe("CreateManagedServiceBinding", func() {
+	Describe("CreateManagedServiceBinding of type key", func() {
```

Comment:
I can see that the `BeforeEach` and `JustBefoereEach` of the describes for type `key` and type `app` are almost identical. I would suggest a test structure as follows:

```
Describe("CreateManagedServiceBinding", func(){
  var message repositories.CreateServiceBindingMessage

  BeforeEach(func(){
    // create the service instance here
  })

  JustBeforeEach(func(){
    // call repo.CreateServiceBinding here using the var'd message
  })

  Describe("type key", func(){
    BeforeEach(fucn(){
      message = // message for type key
    })
  })

  Describe("type app", func(){
    BeforeEach(fucn(){
      message = // message for type app
    })
  })
})
```